pom.xml

```diff
@@ -83,6 +111,11 @@
                     </execution>
                 </executions>
             </plugin>
+<!--            <plugin>-->
+<!--                <groupId>org.flywaydb</groupId>-->
+<!--                <artifactId>flyway-maven-plugin</artifactId>-->
+<!--                <version>${flyway.plugin.version}</version>-->
+<!--            </plugin>-->
```

Comment:
remove if not needed

Reply:
I will remove it in the final commit if not needed

Reply:
Done